lib/shoryuken/middleware/server/auto_retry.rb

```diff
@@ -0,0 +1,43 @@
+module Shoryuken
+  module Middleware
+    module Server
+      class AutoRetry
+        include Util
+
+        def call(worker, queue, sqs_msg, body)
+          yield
+        rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?